src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -40,14 +40,18 @@
     :mention
     (conj
      units
-     [rn/touchable-opacity
```

Comment:
This component is inside the chat `Text` component. While all other inline elements inside the chat `Text` component are of type `Text`, here we are using the `Touchable` component, and it was causing alignment issues.

To fix this issue, I replaced the Touchable with a Text component. However, since this new `Text` component is nested inside another `Text` component, it won't support `padding` and `border-radius`. To create a similar effect to `padding`, I added a single space character on each side. But, with this implementation `border-radius` is still an issue.

I would appreciate your thoughts and suggestions on this.

Reply:
@ajayesivan, my first thoughts were that we're trading one bug for another, so I'm not sure if it's worth the trouble. Have you thought about using the `Pressable` API instead of a `Touchable`? Do you think that could lead to better results and no alignment issues?

Reply:
It's not a perfect solution. But I think it's better than this:
<img src="https://user-images.githubusercontent.com/19279756/233352244-b29a9c9f-34cf-4ba0-ac3d-d62f40332ee4.png" width="400px"/>

Reply:
Indeed, it looks terrible. Are you going to create a separate follow-up issue for the new "bugs" being introduced by this PR?

Reply:
I didn't try `Pressable` but I think it won't make any difference(I tried replacing the touchable with `View`). From my understanding, the root cause of this issue is that each nested `Text` component is not using Flexbox. When we use a `View` or any other element besides Text, it uses Flexbox, which can cause misalignment issues.

Reply:
https://github.com/status-im/status-mobile/issues/15702

Reply:
Thank you @ajayesivan!